docker/docker-compose.yml

```diff
@@ -260,3 +260,31 @@ services:
       - ${MF_COAP_ADAPTER_PORT}:${MF_COAP_ADAPTER_PORT}/tcp
     networks:
       - mainflux-base-net
+
+  twins:
+    image: mainflux/twins:latest
+    container_name: mainflux-twins
+    depends_on:
+      - authn
+      - nats
```

Comment:
please be aware that the twins service depends on a mongo database, what means that either the dependency has to be to the standard mongodb from the writer, or - better, a separate twins-db has to be created and added to the docker-compose file